megamek/src/megamek/common/Entity.java

```diff
@@ -11365,8 +11365,7 @@ public PilotingRollData checkSideSlip(EntityMovementType moveType,
             if (isUsingManAce()) {
                 roll.addModifier(-1, "Maneuvering Ace");
             }
-            if ((getMovementMode() == EntityMovementMode.VTOL) && isMASCUsed()
-                    && hasWorkingMisc(MiscType.F_JET_BOOSTER)) {
+            if (usingJetBooster) {
```

Comment:
Is there a way to have this go into an override of the method in the VTOL class instead? Have it call the base class and then append the "if using jet booster" stuff to the resulting PilotingRollData?

Reply:
It would be possible, but it wouldn't prevent the parameter list from expanding. It wouldn't be an override if it had different parameters, and if the VTOL version had different parameters you would have to conditionally typecast to VTOL to call the right method. I think it's much simpler to leave this code where it is.

Reply:
I thought about it a bit more and came up with a way to simplify it just a bit, taking your suggestion to override the method in VTOL.